contracts/scripts/system_components.json

```diff
@@ -0,0 +1,64 @@
+{ 
+    "CreateCaravan": [
+        "Movable",
+        "Capacity",
+        "CaravanMembers",
+        "Position",
+        "Owner",
+        "ForeignKey"
+    ],
+    "CreateFreeTransportUnit": [
+        "Owner",
+        "Position",
+        "MetaData",
+        "Quantity",
+        "Movable",
+        "ArrivalTime",
+        "Capacity",
+        "QuantityTracker"
+    ],
+    "BuildLabor": [
+        "Labor",
+        "Resource"
+    ],
+    "HarvestLabor": [
+        "Labor",
+        "Resource"
+    ],
+    "AttachCaravan": [
+        "Caravan",
+        "Movable",
+        "OrderId"
+    ],
+    "ClaimFungibleOrder": [
+        "Trade",
+        "Resource"
+    ],
+    "MakeFungibleOrder": [
```

Comment:
"Status" is missing here. Can you double check in all systems that no components are missing?

Reply:
Updated and double checked